validator/src/main.rs

```diff
@@ -2180,7 +2180,7 @@ pub fn main() {
     let restricted_repair_only_mode = matches.is_present("restricted_repair_only_mode");
     let accounts_shrink_optimize_total_space =
         value_t_or_exit!(matches, "accounts_shrink_optimize_total_space", bool);
-    let tpu_use_quic = matches.is_present("tpu_use_quic");
+    let _tpu_use_quic = matches.is_present("tpu_use_quic");
```

Comment:
If we decided to use quic by default. Maybe we should remove the args altogether as well?

Reply:
this is just a test